server/views/partials/varyCallToActions.njk

```diff
@@ -11,13 +11,37 @@
                     attributes: { 'data-qa': 'vary-licence' }
                 }) }}
             {% endif %}
+            {% if callToActions.shouldShowPrintToActivateButton %}
+                {{ govukButton({
+                    text: "View and print licence PDF",
```

Comment:
Doesnt seem like this button prints the PDF? Does it just set it to ACTIVE? Feel like the button should say something more like what it does. Also, probably ought to have an `Are you sure?` screen like the other status changes

Reply:
Haha. I laugh because I talked to Ricky about this twice. I set it to `Set to active` originally. Ricky wanted `Activate licence`. Jo wanted `View and print licence PDF` - even though it doesn't. What it does do is to redirect to the timeline again with the print links enabled. Its one on the list for the review.

Reply:
*sigh*